array_api_compat/dask/array/linalg.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from dask.array.linalg import *
+from ...common import _linalg
+from ..._internal import get_xp
+from dask.array import matmul, tensordot, trace, outer
+from ._aliases import matrix_transpose, vecdot
+
+import dask.array as da
+
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from typing import Optional, Union, Tuple
+    from ...common._typing import ndarray, Device, Dtype
+
+#cross = get_xp(da)(_linalg.cross)
+#outer = get_xp(da)(_linalg.outer)
+EighResult = _linalg.EighResult
+QRResult = _linalg.QRResult
+SlogdetResult = _linalg.SlogdetResult
+SVDResult = _linalg.SVDResult
+qr = get_xp(da)(_linalg.qr)
+#svd = get_xp(da)(_linalg.svd)
```

Comment:
Can you get rid of all the little comment out bits of code like this. Or if one of them actually is useful to keep, add an explanatory comment above it.

Reply:
Sorry, had some code left over from debugging.

Everything should be cleaned up now. 🤞